src/components/NavBar/NavBarProducts/NavBarProducts.jsx

```diff
@@ -54,58 +66,71 @@ const PublishLogo = () => (
   </svg>
 );
 
-const ReplyLogo = () => (
-  <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-    <path d="M16 9.03754H8.9274V16H16V9.03754Z" fill="#132062" />
-    <path fillRule="evenodd" clipRule="evenodd" d="M0 0H13.8055V6.94855H6.83673V13.911L0 13.925V0Z" fill="#FF9B6B" />
-  </svg>
-);
-
 const AnalyzeLogo = () => (
   <svg width="19" height="20" viewBox="0 0 17 18" fill="none" xmlns="http://www.w3.org/2000/svg">
     <path fillRule="evenodd" clipRule="evenodd" d="M7.1336 2.8703C3.28309 2.8703 0.167206 6.03021 0.167206 9.93515C0.167206 13.8401 3.28309 17 7.1336 17C10.9841 17 14.1 13.8401 14.1 9.93515H7.1336V2.8703Z" fill="#F3AFB9" />
     <path fillRule="evenodd" clipRule="evenodd" d="M16 8.00839C16 4.10346 12.8841 0.943542 9.0336 0.943542V8.00839H16Z" fill="#132062" />
   </svg>
 );
 
+const EngageLogo = () => (
+  <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M16 9.03754H8.9274V16H16V9.03754Z" fill="#132062" />
+    <path fillRule="evenodd" clipRule="evenodd" d="M0 0H13.8055V6.94855H6.83673V13.911L0 13.925V0Z" fill="#87C221" />
+  </svg>
+);
+
 const getLogo = (product) => {
   switch (product) {
-    case 'reply':
-      return <ReplyLogo />;
     case 'analyze':
       return <AnalyzeLogo />;
     case 'publish':
       return <PublishLogo />;
+    case 'engage':
+      return <EngageLogo />;
     default:
       return null;
   }
 }
 
-const NavBarProduct = ({ products, activeProduct }) => (
-  <StlyedNavBarProduct>
-    {products.map(product => (
-      <ProductLink
-        active={activeProduct === product}
-        href={`https://${product}.buffer.com`}
-        key={product}
-      >
-        {getLogo(product)}
-        <ProductText>
-          {product}
-        </ProductText>
-      </ProductLink>
-    ))}
-  </StlyedNavBarProduct>
-);
+const NavBarProducts = ({ products, activeProduct }) => {
+  const visibleProducts = products.filter(product => product.visible);
+
+  return (
+    <StlyedNavBarProducts>
+      {visibleProducts.map(({ id, enabled, enableURL, isNew }) => (
+        <ProductLink
+          active={activeProduct === id}
+          href={enabled ? `https://${id}.buffer.com` : enableURL}
+          key={id}
+          id={`product-${id}`}
+        >
+          {getLogo(id)}
+          <ProductText>
+            {id}
+          </ProductText>
+          {isNew && <NewLabel>New!</NewLabel>}
```

Comment:
I wonder if we could add an aria-label here to give a bit more info to our screen reader users. Right now a screen reader reads this as 'engagenew' (https://share.buffer.com/RBuqw2nQ sorry this didn't record my screen reader but I hope it gives a bit of context). We could do an aria-label="Engage is our new product, give it a try!" jajaja not sure 🙈

Reply:
Love this! ❤️I just added it and changed the copy a little to be generic. Going to double-check the copy with Tom to see what we want to have there 🙂 

Thanks so much @gmzjuliana! I have so much to learn from you! ❤️